protocols/raft/src/main/java/io/atomix/protocols/raft/proxy/impl/RetryingRaftProxyClient.java

```diff
@@ -92,7 +92,7 @@ private void execute(RaftOperation operation, int attemptIndex, CompletableFutur
 
   private void scheduleRetry(RaftOperation operation, int attemptIndex, CompletableFuture<byte[]> future) {
     RaftProxy.State retryState = client.getState();
-    scheduler.schedule(delayBetweenRetries, () -> {
+    scheduler.schedule(delayBetweenRetries.multipliedBy(2 ^ attemptIndex), () -> {
```

Comment:
During leader changes, the retrying proxy can often exhaust its requests before a new leader is even elected. Using exponential backoff reduces the frequency of this occurring.

Reply:
Math max with max delay?